app/src/main/java/com/amaze/filemanager/ui/activities/AboutActivity.java

```diff
@@ -235,7 +271,10 @@ public void onClick(View v) {
                 .withAboutSpecial1Description(getString(R.string.amaze_license))
                 .withLicenseShown(true);
 
-        switch (getAppTheme().getSimpleTheme()) {
+        switch (getAppTheme()
+            .getSimpleTheme(
+                (getResources().getConfiguration().uiMode & Configuration.UI_MODE_NIGHT_MASK)
```

Comment:
Deduplicate this, it seems to be copied *a lot* of times.

Reply:
✅ Done